src/main/java/com/puppycrawl/tools/checkstyle/site/ExampleMacro.java

```diff
@@ -0,0 +1,135 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Can we move all our functional classes to `test` folder ?
To avoid dealing with coverage of any sort for them. They are helping classes of execution test class.

Reply:
@romani I spent a couple of hours trying this a few days ago but I couldn't manage to get doxia and plexus find these classes.

Reply:
we will do this https://github.com/orgs/checkstyle/projects/5?pane=issue&itemId=33782286 after functionality is stabilized.
ok to keep for now.